electionleaflets/apps/leaflets/forms.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, **kwargs):
         else:
             self.fields["image"] = S3UploadedImageField(
                 widget=forms.ClearableFileInput(
-                    attrs={"multiple": True, "accept": "image/*",}
+                    attrs={"accept": "image/*",}
```

Comment:
Will this remove the ability to upload more than one leaflet image at a time? Can you check?

Reply:
Multiple uploads still possible. 
<img width="725" alt="Screenshot 2024-09-30 at 9 52 05 AM" src="https://github.com/user-attachments/assets/5010e5b7-5d22-4b6e-8b14-e8ac6dcbac45">

Reply:
Throwing an exception on this was a change made in the release of Django 3.2.19
https://docs.djangoproject.com/en/4.2/releases/3.2.19/
so we can't just pass `"multiple": True` here on the version of Django we're moving to.

This seems to be the recommended way to allow a single form field to submit >1 file now
https://docs.djangoproject.com/en/4.2/topics/http/file-uploads/#uploading-multiple-files
but I'm not 100% clear on whether leaflets actually relies on that.

I'm unclear from the docs whether this code ever did what we thought it did tbh.